portal/views/portal.py

```diff
@@ -144,6 +144,18 @@ def exercise_and_diet():
 def lived_experience():
     return render_template('gil/lived-experience.html', user=current_user())
 
+@portal.route('/david-andrew-story')
```

Comment:
Can I push back on having all these links in the root context?  Perhaps that was requested?  I'd rather see something like a nested /stories/... directory for easier growth down the road.